spec/ssm_config_yml_spec.rb

```diff
@@ -0,0 +1,100 @@
+require "rails_helper"
+
+RSpec.describe SsmConfig do # do, when, it, expect
+  before do
+    stub_const("SsmConfig::CONFIG_PATH", "../fixtures")
+  end
+
+  context "When a YAML file exists" do
+    context "when YAML file is blank" do
+      let(:no_method_error_message) { "undefined method `[]' for false:FalseClass" }
+      context "with 0 depth" do
+        it "raises NoMethodError"
+          expect{SsmConfig.blank}.to raise_error(NoMethodError).with_message(no_method_error_message)
+        end
+      end
+      context "with nontrivial depth" do
```

Comment:
I think we typically have newlines between consecutive groups, context, it... please add a new line here and for some of the other examples.